specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/mfe.json

```diff
@@ -9,6 +9,2478 @@
     "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/registries/{registryName}/codes": {
```

Comment:
This swagger file does not have a globally defined "consumes" and "produces" attributes, nor do we have in these new operations, therefore this is causing our SDK generations to fail.
Could you either
- add produces and consumes in all these operations
- or add a global produces and consumes setting in this swagger file?
Feel free to take either approach you prefer